js/components/utils.js

```diff
@@ -66,36 +63,38 @@ var Utils = {
    *  currentTime: Formatted value of current playhead
    *  totalTime: Formatted value of video duration
    * Live - No DVR
-   *  currentTime: null
-   *  totalTime: null
+   *  currentTime: Empty string
+   *  totalTime: Empty string
    * Live - DVR - useNegativeDvrOffset === true
-   *  currentTime: Formatted value of the negative offset from the live playhead
-   *  totalTime: null
+   *  currentTime: Formatted value of the negative offset from the live playhead. Empty string if playhead is at the Live position
+   *  totalTime: Empty string
    * Live - DVR - useNegativeDvrOffset === false
    *  currentTime: Formatted value of the current playhead relative to max time shift
    *  totalTime: Formatted value of the total duration of the DVR window
+   * NOTE:
+   * Either property can be returned as an empty string if the parameters don't match the requirements.
    *
    * @function getTimeDisplayValues
    * @param {Number} currentPlayhead The current value of the playhead in seconds.
-   * @param {Number} duration The total duration of the video in seconds.
+   * @param {Number} duration The total duration of the video in seconds. Should be -0 or Infinity for Live videos with no DVR.
```

Comment:
is -0 actually a thing or did you mean -1?

Reply:
Yes, this actually what the playhead is set to in these cases. I don't know why it was done like this, but I didn't wan't to risk breaking anything by changing it.

Reply:
Sounds good.